### PR TITLE
Changed bibtex checkboxes to radios in cleanup dialog

### LINKED ISSUE #10583: Change checkbox to a radio button in Cleanup for converting to biblatex/bibatex
**Is your suggestion for improvement related to a problem? Please describe.**
Currently, two checkboxes are displayed for "Convert to BibTeX" and "Convert to biblatex" but one can only choose one. 

**Describe the solution you'd like**
The checkboxes should be replaced by radio buttons with a Toggle Group

**Additional context**

grafik

### PR DESCRIPTION
**Pull request by suxiaocha123**
Resolves #10583

### Mandatory checks

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by calixtus** [comment]
Please link to an issue and elaborate your PR description

**Comment by suxiaocha123** [comment]
I'm sorry, it has been modified now.

**Comment by suxiaocha123** [comment]
> Please link to an issue and elaborate your PR description

I'm sorry, it has been modified now

File: src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml [hunk 1]
```diff
@@ -30,8 +30,10 @@
           <Insets left="20.0" />
        </VBox.margin>
       </VBox>
-      <CheckBox fx:id="cleanUpBiblatex" text="%Convert to biblatex format (e.g., store publication date in date field)" />
-      <CheckBox fx:id="cleanUpBibtex" text="%Convert to BibTeX format (e.g., store publication date in year and month fields)" />
+      <ToggleGroup fx:id="conversionToggleGroup">
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml:33]: You need to define the toggle group first, like the timestampCleanup

**Review by Siedlerchr — CHANGES_REQUESTED** [review]
Tests are failing. You would have seen this error as well if you started JabRef and opened the CleanupdDialog

File: src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml (hunk 1 above)
Comment (suxiaocha123) [reply src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml:33]: I'm sorry. I will revise it as soon as possible

**Comment by calixtus** [comment]
We noticed a problem with the issue:
- What if the user clicks on the converte checkbox by accident, but does not want to convert anything? The user cant unclick the radio button, it stays selected.
- Alternative A: Add a third radio button "do not convert to bibtex/biblatex"
- Alternative B: Keep as mutually exclusive checkboxes, that can be unclicked
- @Siedlerchr and me decided for Solution B, since solution A would show a visiual indication, that sthg will be changed although nothing will be done.

**Comment by calixtus** [comment]
And I just saw, that the logic is already implemented here:
https://github.com/JabRef/jabref/blob/ec81fc65253196716428340addbfafcddd3d2b55/src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.java#L74-L97

I believe this PR is therefor obsolete. Im very sorry @suxiaocha123 that you have invested time into this non-issue. Maybe another 'good first issue' is something for you?

**Pull request closed without merging**